### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by pavlosoletskyy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
-  - An `EnglishGreeting` that greets "world" or a specific person in English.
-  - A `FrenchGreeting` that greets "tout le monde" or a specific person in French.
-  - A `SpanishGreeting` that greets "mundo" or a specific person in Spanish.
+In this task, you're provided with a base class named `HelloWorldAnonymousClasses`. It contains an interface `HelloWorldGreeting` with two methods: `greet()` and `greetSomeone(someone: String)`. 
 
-- Each greeting class should have a `greet()` method that returns a general greeting to the world, and a `greetSomeone(someone: String)` method that returns a personalized greeting to the specified person.
+**Your goal:** implement the `sayHello(names: List<String>): List<String>` method using anonymous classes to provide greetings in **English, French,** and **Spanish**.
 
-- In the `sayHello` method, create a list of these greeting objects and use it to compile a list of general greetings from each language (e.g. "Hello world").
+## Instructions:
```
Comment (alenatovstukha) [inline README.md:8]: ## Instructions

File: README.md [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
+    
+ 5. Then, for each name in the `names` parameter, generate personalized greetings in all three languages and append them to the list of general greetings.
+    
+ 6. The `sayHello` function should return a list of strings containing both the initial general greetings (e.g., “Hello world”) and all the personalized greetings.
+    
 
-- The `sayHello` function should return a list of strings containing both the initial general greetings (e.g. "Hello world") and all the personalized greetings.
-
-Example of how your implementation should work:
+## Example of how your implementation should work:
```
Comment (alenatovstukha) [inline README.md:26]: ## Example of How Your Implementation Should Work

File: README.md [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
+ 1. Begin by creating three anonymous classes within the `sayHello` method, each implementing the `HelloWorldGreeting` interface.
+    
+ 2. Each class should offer its own implementation of the `greet()` and `greetSomeone(someone: String)` methods:
+    -   An `EnglishGreeting` that greets “world” or a specific person in English.
+    -   A `FrenchGreeting` that greets “tout le monde” or a specific person in French.
+    -   A `SpanishGreeting` that greets “mundo” or a specific person in Spanish.
+      
+ 3. Each greeting class should include a `greet()` method that returns a general greeting to the world and a `greetSomeone(someone: String)` method that returns a personalized greeting to the specified person.
+    
+ 4. In the `sayHello` method, create a list of these greeting objects and utilize it to compile a list of general greetings from each language (e.g., “Hello world”).
```
Comment (alenatovstukha) [inline README.md:19]: 4. In the `sayHello` method, create a list of these greeting objects and utilize it to compile a list of general greetings from each language (e.g., `Hello world`).

File: README.md [hunk 4]
```diff
@@ -1,23 +1,29 @@
...
+**Your goal:** implement the `sayHello(names: List<String>): List<String>` method using anonymous classes to provide greetings in **English, French,** and **Spanish**.
 
-- In the `sayHello` method, create a list of these greeting objects and use it to compile a list of general greetings from each language (e.g. "Hello world").
+## Instructions:
 
-- Then, for each name in the `names` parameter, generate personalized greetings in all three languages and append them to the list of general greetings.
+ 1. Begin by creating three anonymous classes within the `sayHello` method, each implementing the `HelloWorldGreeting` interface.
+    
+ 2. Each class should offer its own implementation of the `greet()` and `greetSomeone(someone: String)` methods:
+    -   An `EnglishGreeting` that greets “world” or a specific person in English.
```
Comment (alenatovstukha) [inline README.md:13]: `world`

File: README.md [hunk 5]
```diff
@@ -1,23 +1,29 @@
...
+    -   A `FrenchGreeting` that greets “tout le monde” or a specific person in French.
```
Comment (alenatovstukha) [inline README.md:14]: `tout le monde`

File: README.md [hunk 6]
```diff
@@ -1,23 +1,29 @@
...
+    -   A `SpanishGreeting` that greets “mundo” or a specific person in Spanish.
```
Comment (alenatovstukha) [inline README.md:15]: `mundo`

**Review by alenatovstukha — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alenatovstukha) [reply README.md:26]: або просто Example

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**